scss/_alert.scss

```diff
@@ -5,7 +5,7 @@
 .alert {
   padding: $alert-padding;
   margin-bottom: $spacer-y;
-  border: 1px solid transparent;
+  border: $alert-border-width solid transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.